src/index.ts

```diff
@@ -1565,6 +1759,8 @@ class RawRemarkable implements RawRemarkableApi {
       return loaded;
     } else if (documentContent.guardAssert(loaded)) {
       return loaded;
+    } else if (templateContent.guardAssert(loaded)) {
+      return loaded;
```

Comment:
`guardAssert` will raise an error for false, so this shouldn't work. `documentContent` is pretty forgiving, so I would move `templateContent` before `documentContent` and change it to a standard `guard` instead of the `guardAssert`.

There might a better way to handle this, but I'll probably do it on top of this commit.

> Note to myself in the future, should be able to loop through the validators with a guard assert and a try, on failure, append errors to an array and rethrow the concatenation of those errors if none-pass.